### DISCUSSION TITLE
Server responded with a status of 500

### DISCUSSION THREAD

**Discussion by eurunuela** [vercel/community#2853] [Help] [answered]
### Summary

I changed the region in Functions as suggested on this forum, and my site deploys but I get an unexpected server error when visiting.

### Example

See: https://www.enekourunuela.com/

### Steps to Reproduce

Change deployment region and deploy.

Edit: the deployment seems to be working but the site can no longer download data on load and shows the following error in the console:

```
Failed to load resource: the server responded with a status of 500 ()
```

It seems to happen with my previous production builds.

**Comment by smaeda-ks** [comment, marked as answer]
@eurunuela you can always check runtime error logs in your Vercel dashboard:
https://vercel.com/docs/concepts/observability/runtime-logs

From there, you can see it's failing at here:
https://github.com/eurunuela/eurunuela.github.io/blob/main/app/routes/cards/conferencesCard.tsx#L36-L38

Make sure that `abstract["awards"]` data is actually an Array type, not String. Otherwise, `.join` fails in such an error

**Reply by eurunuela** [reply]
> @eurunuela you can always check runtime error logs in your Vercel dashboard:
> 
> https://vercel.com/docs/concepts/observability/runtime-logs
> 
> 
> 
> From there, you can see it's failing at here:
> 
> https://github.com/eurunuela/eurunuela.github.io/blob/main/app/routes/cards/conferencesCard.tsx#L36-L38
> 
> 
> 
> Make sure that `abstract["awards"]` data is actually an Array type, not String. Otherwise, `.join` fails in such an error

Gosh... such a silly mistake. I'm sorry about that. Thank you so much for your help!
